source/pasta/brusselSproutGnocchi.rst

```diff
@@ -0,0 +1,48 @@
+Gnocchi with Brussel Sprouts, Garlic, and a boat-load of butter
+===============================================================
+
+Details
+-------
+
+* Number of servings: 6
+* Cook time: 20 minutes
+
+Ingredients
+-----------
+
+* 1 lb Brussel sprouts, trimmed and halved so they have a big flat face
+* 500g gnocchi (or any type of pasta)
+* At least 4 cloves of garlic (prefferably more), finely minced
+* The juice and zest of 1 lemon (zest less important than juice)
+* 6 tbsp butter
+* Chilli flakes
+
+Optional:
+
+* Parmesan
+* 1/2 tsp honey
+
+Equipment
+---------
+
+* A saucepan for cooking the gnocchi/pasta
+* The widest frying pan you have, prefferably with high sides
+* A strainer
+
+Steps
+-----
+
+1. Put salted water on to boil
+2. Put a decent covering of olive oil in the frying pan, and place the sprouts in it, flat-side down.
+3. Scatter lemon-zest on top, put the heat on medium high, cover the pan with a lid and cook for 3-5 minutes, until the sprouts are browned and are soft, remove from pan.
+4. Put the gnocchi in the boiling water, for 2-3 minutes, just until they float, then drain.
+5. Add the drained gnocchi to the frying pan (you may need to add a little more oil) and cook for 2-3 minutes, until they start to crisp up.
+6. Add back in the sprouts, along with the garlic, chilli flakes, butter, salt and pepper, cook until the butter starts to brown.
+7. Add the lemon juice, honey if using, and a splash of pasta water, stir everything up to combine, and serve with parmesan.
```

Comment:
Add the lemon juice, honey if using, and a splash of pasta water, stir to combine. Serve with parmesan.